### PR TITLE
add PDS template 4.49

### LINKED ISSUE #725: Add PDT 4.49
To encode ensemble information in metadata of aerosol fields in GRIB2 data, [PDT 4.49](https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_temp4-49.shtml) should be added in pdstemplates.F90.

### PR DESCRIPTION
**Pull request by edwardhartnett**
Fixes #725

### CODE REVIEW

File: src/pdstemplates.F90 [hunk 1]
```diff
@@ -341,6 +341,15 @@ module pdstemplates
...
 
+  !
+  !    PDT 4.49
+  !
+  data templates(44)%template_num /49/     ! Individual Ensemble Forecast, Control and Perturbed, 
+  data templates(44)%mappdslen /29/        ! at a horizontal level or in a horizontal layer at a
+  data templates(44)%needext /.false./     ! point in time for Optical Properties of Aerosol for 
+  data (templates(44)%mappds(j), j = 1, 29) &  ! Optical Properties of Aerosol
+       /1, 1, 2, 1, -1, -4, -1, -4, 1, -1, -4, -1, -4, 1, 1, 1, 2, 1, 1, -4, 1, -1, -4,  &
+       1, -1, -4, 1, -1, 1/
```
Comment (lipan-NOAA) [inline src/pdstemplates.F90]: line 352 1, -1, -4, 1, -1, 1 or 1, -1, -4, 1, 1, 1  ?

Comment (WenMeng-NOAA) [reply src/pdstemplates.F90]: @edwardhartnett From Li's testing and my testing, the line 352 is "1, -1, -4, 1, 1, 1/"

Comment (edwardhartnett) [reply src/pdstemplates.F90]: Good catch. So the field Perturbation number can never be negative?

**Review by lipan-NOAA — APPROVED** [review]

**Comment by edwardhartnett** [comment]
@Hang-Lei-NOAA or @AlexanderRichert-NOAA I need one of you to approve this PR before it can be merged...

**Review by Hang-Lei-NOAA — APPROVED** [review]

**Pull request merged**